atomata/capstone_WebClient
# Enhancement of the selection and login page

### Summary 
-made changes to the selection page as per stakeholder request.
-removed the white space on the login page.

### How to test?
1. Check if you see a white space at the bottom of the login page anymore. (Test on different browsers)
2. Check if the UI of the selection page is according to stakeholder's request (All the Boxes edges are curved and its color is like the box in the workbench page, background color like workbench page)

### Checks
